lib/commandCenter/commandCenter.js

```diff
@@ -1,39 +1,39 @@
-var async = require('async');
-var zlib = require('zlib');
-var fs = require('fs');
-var path = require('path');
-var util = require('util');
-var functionArguments = require('function-arguments');
-
-var EventEmitter = require('events').EventEmitter;
-var ResponseCache = require('./responseCache').ResponseCache;
-var httpBatchHandler = require('./httpBatchHandler');
-
-var mage;
-var logger;
-var State;
-
-var DEFAULT_RESPONSE_CACHE_TTL = 3 * 60;  // cache lifetime of the command response in seconds
-var COMPRESSION_THRESHOLD = 4096;         // only gzip if the response is at least this many bytes uncompressed
-var COMMANDS_DIR_NAME = 'usercommands';     // user command directory inside of module file space
-var COMMAND_PATH_SPLITTER = '.';                // what separates the module name and command name
-var CONTENT_TYPE_JSON = 'application/json; charset=UTF-8';
+'use strict';
 
-exports = module.exports = new EventEmitter();
+const async = require('async');
+const zlib = require('zlib');
+const fs = require('fs');
+const path = require('path');
+const functionArguments = require('function-arguments');
+
+const EventEmitter = require('events').EventEmitter;
+const ResponseCache = require('./responseCache').ResponseCache;
+const httpBatchHandler = require('./httpBatchHandler');
+
+let mage;
+let logger;
+let State;
 
-function UserCommandSetupError(message, details) {
-	Error.captureStackTrace(this, this.constructor);
-	Object.assign(this, details);
+const DEFAULT_RESPONSE_CACHE_TTL = 3 * 60;  // cache lifetime of the command response in seconds
+const COMPRESSION_THRESHOLD = 4096;         // only gzip if the response is at least this many bytes uncompressed
```

Comment:
probably not the scope of this PR but making some of these configurable might be nice?